DataAccessGeneration.XUnitTest/GenerateInterface.cs

```diff
@@ -18,7 +18,25 @@ public static void CorrectTypeIEnumerable()
                 }
 
             },
-            "Repo", "Proc_Result", new List<string>(){"UDFProperty"}, new List<UserDefinedTableRowDefinition>()
+            "Repo", "Proc_Result", new FakeDataLookup()
+            {
+                GetUserDefinedTypeData = new UserDefinedTableGrouping()
+                {
+                    UDTTypeName = "UDFProperty",
+                    Rows = new List<UserDefinedTableRowDefinition>()
+                    {
+                        new UserDefinedTableRowDefinition()
+                        {
+                            TypeName = "uniqueidentifier",
+                            TableTypeName = "UDFProperty",
+                            ColumnName = "InnerColumnName",
+                            SchemaName = "DBO"
+                        }
+                    }
+                }
+            });
+        /*
```

Comment:
Necessary to keep the commented out code?

Reply:
Nope. Fixing